packages/rspack/src/fileSystem.ts

```diff
@@ -12,33 +13,53 @@ function createThreadsafeNodeFSFromRaw(
 	fs: typeof import("fs")
 ): ThreadsafeWritableNodeFS {
```

Comment:
Perhaps it could be saved at initialization instead of during function calls? Otherwise, `bind` and `promisify` will be executed every time the function is called.

Reply:
done